app/dashboard/templates/dashboard.html

```diff
@@ -32,24 +32,42 @@
       {% include 'shared/nav.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col-12 col-lg-3 col-xl-2" id="sidebar_container" style="background: #F2F6F9;">
+      <div class="col-12 col-lg-3 col-xl-2 invisible" id="sidebar_container" style="background: #F2F6F9;">
         {% include 'shared/current_balance.html' %}
         {% include 'shared/sidebar_search.html' %}
       </div>
 
       <div class="col-12 col-lg-9 col-xl-10 body">
         <div class="container-fluid" id="bounties">
-          {% include 'shared/search_bar.html' %}
-          <div class="row mt-2 mb-2">
-            <div class="col-12 offset-md-1 col-md-10 title-row">
-              <div class="col-12 col-sm-7 col-md-5">
-                <span id="filter" class="font-title"></span>
-                <span id="matches" class="font-subheading"></span>
-              </div>
-              <div class="col-sm-5 col-md-7 bounty-info" id="funding-info">
+          <div id="dashboard-title" class="hidden">
+            {% include 'shared/search_bar.html' %}
+            <div class="row mt-2 mb-2">
+              <div class="col-12 offset-md-1 col-md-10 title-row">
+                <div class="col-12 col-sm-7 col-md-5">
+                  <span id="filter" class="font-title"></span>
+                  <span id="matches" class="font-subheading"></span>
+                </div>
+                <div class="col-sm-5 col-md-7 bounty-info" id="funding-info">
+                </div>
               </div>
             </div>
           </div>
+          <div id="onboard-dashboard" class="row hidden font-body">
+            <div class="col-12 offset-md-1 col-md-9">
+              <h2 class="font-title">Welcome to Gitcoin!</h2>
+              <h3 class="font-subheader">We've found bounties that match your interest and experience</h3>
```

Comment:
We'll want to wrap all of the copy throughout this file with `{% trans %}` and `{% blocktrans %}` where applicable.